### ISSUE TITLE
508-defect-3 [COGNITION]: The supporting evidence page should be split to reduce cognitive load

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#19733]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh_

## User Story or Problem Statement

As a user with cognitive considerations, I want to be able to work on tasks one at a time.

## Details

The supporting evidence page currently uses a wizard to dynamically show and hide content to the user. This may be difficult for users with cognitive considerations, screen reader users, and keyboard users for the following reasons:
- Expanding and collapsing large sections can be distracting
- Increased cognitive load of having to complete _multiple_ tasks on a single page
- The more fields there are, the more tedious it becomes to navigate for keyboard and screen reader users
- It reduces a sense of progression

For the above reasons and more, it would be better to [split these apart into two separate pages following the method of one thing per page (as seen on gov.uk and nhs)](https://www.smashingmagazine.com/2017/05/better-form-design-one-thing-per-page/).

## Acceptance Criteria

- [ ] Yes/no question and uploading supporting evidence are split onto their own pages

## Proposed Solution 

Remove the wizard in favor of splitting the conditional into two separate pages. The user can answer the yes no question first, continue to the next page, and then upload information. Instructions will be easier to read, a sense of progression will be created, cognitive load will be reduced, and there will be less expanding/collapsing movement on the page.

## Screenshots or Trace Logs

<img width="1792" alt="Screen Shot 2021-01-26 at 10 08 18 AM" src="https://user-images.githubusercontent.com/14154792/105863464-ebd59480-5fbe-11eb-9d92-637d5ded31be.png">

**Comment by Mottie** [comment]
Oh, but the alert under the "No" choice would get read out properly until the `selectedProps` code fix (adds dynamic `aria-describedby` attributes) has been merged in.

**Comment by joshkimux** [comment]
<img width="1208" alt="Screen Shot 2021-04-12 at 11 58 11 AM" src="https://user-images.githubusercontent.com/14154792/114424684-5cd02580-9b86-11eb-8825-ae2d73675ca8.png">

Confirmed this page has been split. One thing per page is beautiful! 🥳
